src/Tuura/Boolean.hs

```diff
@@ -1,6 +1,9 @@
+{-# LANGUAGE FlexibleInstances, OverlappingInstances, IncoherentInstances #-}
```

Comment:
Perhaps a `prettyLiteral` function like for the CNF and DNF types is what is needed.